packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewer.js

```diff
@@ -1,112 +1,129 @@
 import { Box } from 'grommet'
-import PropTypes from 'prop-types'
-import { useRef } from 'react'
-import styled, { css } from 'styled-components'
+import { arrayOf, bool, func, number, shape, string } from 'prop-types'
+import { useEffect } from 'react'
 
-import SVGContext from '@plugins/drawingTools/shared/SVGContext'
-import InteractionLayer from '../InteractionLayer'
 import ZoomControlButton from '../ZoomControlButton'
-import locationValidator from '../../helpers/locationValidator'
 
-const PlaceholderSVG = styled.svg`
-  background: no-repeat center / cover url('https://static.zooniverse.org/www.zooniverse.org/assets/fe-project-subject-placeholder-800x600.png');
-  touch-action: pinch-zoom;
-  max-width: ${props => props.$maxWidth || '100%'};
-  ${props => props.$maxHeight && css`max-height: ${props.$maxHeight};`}
-`
-const SVGImageCanvas = styled.svg`
-  overflow: visible;
-`
+import VisXZoom from '../SVGComponents/VisXZoom'
+
+import SingleImageCanvas from './SingleImageCanvas'
+
+const DEFAULT_HANDLER = () => true
+const DEFAULT_ZOOM_CONFIG = {
+  direction: 'both',
+  maxZoom: 10,
+  minZoom: 0.1,
+  onWheelThrottleWait: 100,
```

Comment:
This throttles the mouse wheel zooming. `100` feels better to me, but open to suggestion!